src/main/java/org/jabref/gui/actions/JabRefAction.java

```diff
@@ -42,7 +55,15 @@ private String getActionName(Action action, Command command) {
         if (command.getClass().isAnonymousClass()) {
             return action.getText();
         } else {
-            return command.getClass().getSimpleName();
+            String ans = command.getClass().getSimpleName();
+            if (ans.contains("OldDatabaseCommandWrapper")) {
+                OldDatabaseCommandWrapper tmp = (OldDatabaseCommandWrapper) command;
+                return tmp.getActions().toString();
+            } else if (ans.contains("EditAction")) {
+                return command.toString();
+            } else {
+                return command.getClass().getSimpleName();
+            }
```

Comment:
I have been testing this method and there is a small bug in some `EditAction` cases (it shows `org.jabref.gui.entryeditor.SourceTab$EditAction@56048e40FromMenu`). Due to there are still many classes that use `Actions` and `OldDatabaseCommandWrapper`, it is a bit complicated to deal with all of them, but I think that something like this should work...

```
} else {
    String commandName = command.getClass().getSimpleName();
    if (command instanceof OldDatabaseCommandWrapper || commandName.equals("EditAction")) {
        return action.toString();
    } else {
        return commandName;
    }
}
```
Just a proposal, please adapt it in your own way :"D